solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -1006,23 +1018,35 @@ contract LiquidityProtection is TokenHandler, Utils, Owned, ReentrancyGuard, Tim
      * @param _value           ETH amount to add
      */
     function addLiquidity(
-        ILiquidityPoolV1Converter _converter,
+        ILiquidityPoolConverter _converter,
         IERC20Token _reserveToken1,
         IERC20Token _reserveToken2,
         uint256 _reserveAmount1,
         uint256 _reserveAmount2,
         uint256 _value
     ) internal {
-        IERC20Token[] memory reserveTokens = new IERC20Token[](2);
-        uint256[] memory amounts = new uint256[](2);
-        reserveTokens[0] = _reserveToken1;
-        reserveTokens[1] = _reserveToken2;
-        amounts[0] = _reserveAmount1;
-        amounts[1] = _reserveAmount2;
-
         // ensure that the contract can receive ETH
         updatingLiquidity = true;
-        _converter.addLiquidity{ value: _value }(reserveTokens, amounts, 1);
+
+        if (isStandardPool(_converter)) {
```

Comment:
this needs to be removed, right?

Reply:
@yudilevi : Yes, this is part of the task on Trello.